unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/Environment.cs

```diff
@@ -10,6 +10,7 @@ public class Environment
     {
         public static readonly Environment i = new Environment();
 
+        public WorldState worldState;
```

Comment:
can we make it readonly or convert it into a property so we avoid de possibility being set by mistake?

Reply:
I'll try and add an `Initialize` method so this is possible. In any case, the `readonly` pattern will dissappear in the short term because I found it to be not very viable to be able to mock systems on demand, its just too restrictive.

Reply:
Done